flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
+  "automerge-flathubbot-prs": true,
```

Comment:
I'd prefer if you don't do this. Builds gets pushed without any testing and may result in breakage